src/MainUtils.cpp

```diff
@@ -83,6 +83,11 @@ llvm::cl::opt<bool> useOnnxModelTypes("useOnnxModelTypes",
     llvm::cl::desc("use types and shapes from ONNX model"),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<string> shapeInformation("shapeInformation",
+    llvm::cl::desc("Custom shapes for the inputs of the ONNX model"),
+    llvm::cl::value_desc("<input_id:dim,dim,dim|input_id:dim,dim,dim>"),
+    llvm::cl::cat(OnnxMlirOptions));
```

Comment:
please add a sentence to explain what the input_id is, whether dim are only positive numbers (zero allowed???)

Reply:
Done. Thanks for your comments!